<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QMathMatrixUi</class>
 <widget class="QDialog" name="QMathMatrixUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="rowsL" >
       <property name="toolTip" >
        <string>Number of rows</string>
       </property>
       <property name="text" >
        <string>&amp;Rows:</string>
       </property>
       <property name="buddy" >
        <cstring>rowsSB</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rowsSB" >
       <property name="toolTip" >
        <string>Number of rows</string>
       </property>
       <property name="buttonSymbols" >
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="maximum" >
        <number>511</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="columnsL" >
       <property name="toolTip" >
        <string>Number of columns</string>
       </property>
       <property name="text" >
        <string>&amp;Columns:</string>
       </property>
       <property name="buddy" >
        <cstring>columnsSB</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="columnsSB" >
       <property name="toolTip" >
        <string>Number of columns</string>
       </property>
       <property name="buttonSymbols" >
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="maximum" >
        <number>511</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="EmptyTable" name="table" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Resize this to the correct table dimensions</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="alignmentG" >
     <property name="title" >
      <string>Alignment</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QComboBox" name="valignCO" >
        <property name="toolTip" >
         <string>Vertical alignment</string>
        </property>
        <item>
         <property name="text" >
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Middle</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Bottom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="valignLA" >
        <property name="text" >
         <string>&amp;Vertical:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>valignCO</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="halignED" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Horizontal alignment per column (l,c,r)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="widthLA_2" >
        <property name="text" >
         <string>&amp;Horizontal:</string>
        </property>
        <property name="buddy" >
         <cstring>halignED</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_helpers.h</include>
 </includes>
 <customwidgets>
  <customwidget>
   <class>EmptyTable</class>
   <extends>QWidget</extends>
   <header>emptytable.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rowsSB</tabstop>
  <tabstop>columnsSB</tabstop>
  <tabstop>valignCO</tabstop>
  <tabstop>halignED</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
